WordPress-iOS-Shared/Core/WPStyleGuide.m

```diff
@@ -141,6 +141,11 @@ + (UIFont *)tableviewSectionHeaderFont
     return [WPFontManager systemRegularFontOfSize:13.0];
 }
 
++ (UIFont *)tableviewSectionFooterFont
+{
+	return [WPFontManager systemRegularFontOfSize:13.0];
```

Comment:
`WPTableViewSectionHeaderFooterView` was using `[WPStyleGuide subtitleFont]`, which is size 12 not 13. Did you change the size on purpose? Consistency with the header?